plugins/rbac-backend/src/file-permissions/csv-file-watcher.ts

```diff
@@ -399,10 +378,7 @@ export class CSVFileWatcher {
           roleMetadata.roleEntityRef,
         );
 
-        await this.enforcer.addOrUpdateGroupingPolicy(
-          groupPolicy,
-          roleMetadata,
-        );
+        await this.enforcer.addGroupingPolicy(groupPolicy, roleMetadata);
```

Comment:
It looks like on start up, legacy roles that have a matching role in the CSV file are not updated until a change is made to that role.

Reply:
I think I handled this comment.